src/OpenTelemetry.Exporter.Prometheus/README.md

```diff
@@ -7,17 +7,88 @@
 
 * [Get Prometheus](https://prometheus.io/docs/introduction/first_steps/)
 
-## Installation
+## Steps to enable OpenTelemetry.Exporter.Prometheus
+
+### Step 1: Install Package
 
 ```shell
 dotnet add package OpenTelemetry.Exporter.Prometheus
 ```
 
+### Step 2: Configure OpenTelemetry MeterProvider
+
+* When using OpenTelemetry.Extensions.Hosting package on .NET Core 3.1+:
+
+    ```csharp
+    services.AddOpenTelemetryMetrics(builder =>
+    {
+        builder.AddPrometheusExporter();
+    });
+    ```
+
+* Or configure directly:
+
+    Call the `AddPrometheusExporter` `MeterProviderBuilder` extension to
+    register the Prometheus exporter.
+
+    ```csharp
+    using var meterProvider = Sdk.CreateMeterProviderBuilder()
+        .AddPrometheusExporter()
+        .Build();
+    ```
+
+### Step 3: Register ASP.NET Core Prometheus Scraping Middleware
```

Comment:
Isn't this optional?

Reply:
This PR changes the behavior a bit.

* .NET Framework: It works as it did before. HttpListener is automatically started.

* .NET/Core: HttpListener is NOT automatically started. Users must configure middleware or explicitly set the listener to start.

The reason being, HttpListener goes outside of ASP.NET Core: https://docs.microsoft.com/en-us/dotnet/api/system.net.httplistener?view=net-5.0#httpsys What I'm worried about is customers relying on Kestrel pipeline for filtering, SSL, origin detection, pathBase, whatever, and then the HttpListener circumventing that.

I did update the README to try and make this more clear.